reactive/hive.py

```diff
@@ -1,55 +1,52 @@
-from charms.reactive import when, when_not, when_none
-from charms.reactive import set_state, remove_state
+# pylint: disable=unused-argument
+from charms.reactive import when, when_not, when_not_all
+from charms.reactive import is_state, set_state, remove_state
 from charmhelpers.core import hookenv
-from charms.hive import Hive
+from charms.layer.apache_hive import Hive
 from charms.layer.hadoop_client import get_dist_config
-from charms.reactive.helpers import data_changed
-
-
-@when_not('hadoop.ready')
-def report_blocked():
-    hookenv.status_set('blocked', 'Waiting for relation to Hadoop Plugin')
 
 
 @when('hadoop.ready')
```

Comment:
Do we really need to wait for hadoop.ready here?  Can't we at least install without hadoop?